src/label_extraction.py

```diff
@@ -535,7 +535,7 @@ def _run_test_suite_no_split(data_tsv: Path):
         check_is_independent(train_baseline_tsv, test_baseline_tsv)
 
 
-def _run_test_suite_multiple_splits(data_tsv: Path):
+def _run_test_suite_multiple_splits(data_tsv: Path, n_splits: int):
```

Comment:
I'm probably missing something here, but where is `n_splits` actually used in this function ?

Reply:
In line 539. But you're right, I suspect this line to be superfluous. Let me check!

Reply:
Indeed... I must be getting old and need glasses 😓